Makefile

```diff
@@ -12,7 +13,20 @@ enter_docker_image: build_docker_image
 create_environment:
 	conda create -n $(ENVIRONMENT_NAME) python=3.10 pip
 	conda run --no-capture-output -n $(ENVIRONMENT_NAME) python -m pip install uv==0.2.5
-	conda run --no-capture-output -n $(ENVIRONMENT_NAME) uv pip install -c constraints.txt -e fme[dev]
+	conda run --no-capture-output -n $(ENVIRONMENT_NAME) uv pip install -c constraints.txt -e .[dev]
 
 test:
 	pytest --durations 20 .
+
+# For maintainer use only
+# requires fme[deploy] to be installed
+
+build_pypi:
+	rm -rf dist
+	python -m build
+
+deploy_pypi: build_pypi
```

Comment:
User needs an API key to push, so it's safe to include in make target (i.e., can't accidentally push without access, and pushing same version fails)